index.html

```diff
@@ -53,95 +56,108 @@ <h1 class="first-introduction">I'm Raphael.<br /> I am so glad to see you!</h1>
-                    <li><label class="tool-press">JAVASCRIPT</label></li>
-                </ul>
-                <button class="see-project">See Project</button>
             </div>
         </div>
         <!-- End of Project 3 -->
 
         <!-- Start of Project 4 -->
         <div id="tonic-project">
             <div class="data">
-                <img src="./images/snapshots/Snapshoot Portfolio 4.png" alt="Tonic project snapshot">
-                <h2 class="project-title">Multi-Post <br />Stories</h2>
-                <div class="work-nature">
-                    <label class="canopy">CANOPY</label>
-                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
-                    <label class="backend"> Back End Dev</label>
-                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
-                    <label class="year">2015</label>
+                <div class="image-div"><img src="./images/snapshots/Snapshoot Portfolio 4.png" alt="Tonic project snapshot"></div>
+                <div class="desktop-image-div"><img class="desktop-snapshop" src="./images/desktop/Snapshoot-Desktop4.png" alt="Tonic project snapshot"></div>
+                <div class="project-content-4">
+                    <h2 class="project-title">Multi-Post <br />Stories</h2>
+                    <div class="work-nature">
+                        <label class="canopy">CANOPY</label>
+                        <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                        <label class="backend"> Back End Dev</label>
+                        <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                        <label class="year">2015</label>
+                    </div>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="tools">
+                        <li><label class="tool-press">HTML</label></li>
+                        <li><label class="tool-press">CSS</label></li>
+                        <li><label class="tool-press">JAVASCRIPT</label></li>
+                    </ul>
+                    <button class="see-project">See Project</button>
                 </div>
-                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul class="tools">
-                    <li><label class="tool-press">HTML</label></li>
-                    <li><label class="tool-press">CSS</label></li>
-                    <li><label class="tool-press">JAVASCRIPT</label></li>
-                </ul>
-                <button class="see-project">See Project</button>
             </div>
         </div>
         <!-- End of Project 4 -->
     </div>
+
     <section id="skills-page">
```

Comment:
- [ ] As demanded by the project requirements, you must replicate the design as much as possible (e.g., font, colors, images, text, margins), keeping this in mind, kindly consider the following from the image below.

![image](https://user-images.githubusercontent.com/94551484/229281270-8239d944-fb70-4260-8b42-b305a0eb179a.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/229281307-afead0fd-bfd6-4bb3-b2ed-97de15c76452.png)